attestation-service/src/token/ear_default_policy.rego

```diff
@@ -86,7 +86,33 @@ else := 3 if {
 	input.snp.policy_migrate_ma == 0
 }
 
-##### TDX TODO
+##### TDX
+executables := 3 if {
+	# Check the kernel, initrd, and cmdline (including dmverity parameters) measurements
+	# TODO: add individual CCEL measurements from input.tdx.ccel instead
+	input.tdx.quote.body.rtmr_1 in data.reference.rtmr_1
+	input.tdx.quote.body.rtmr_2 in data.reference.rtmr_2
+}
+
+hardware := 2 if {
+	# Check the quote is a TDX quote signed by Intel SGX Quoting Enclave
+	input.tdx.quote.header.tee_type == "81000000"
+	input.tdx.quote.header.vendor_id == "939a7233f79c4ca9940a0db3957f0607"
+
+	# Check TDX Module version and its hash. Also check OVMF code hash.
+	input.tdx.quote.body.mr_seam in data.reference.mr_seam
+	input.tdx.quote.body.tcb_svn in data.reference.tcb_svn
+	input.tdx.quote.body.mr_td in data.reference.mr_td
```

Comment:
Following https://github.com/confidential-containers/trustee/blob/main/attestation-service/docs/parsed_claims.md#intel-tdx, seems that we can also add
`tdx.quote.body.mrsigner_seam` also?

Reply:
I left that out on purpose for now since that's all zeros for Intel's TDX Module. Do you think it would be good to add a check for zero here?

Reply:
As a sample policy, it is ok imo.

Reply:
OK as it is now?